### PR TITLE
feat(Resource-status): Add Performance page button

### PR DESCRIPTION
**Pull request by jdelpierre**
Add a new button to access performance page from resource status detail panel, graph section.

Refs: MON-10900

Screenshot with the button on the right panel
image

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/Resources/translatedLabels.ts [hunk 1]
```diff
@@ -190,6 +190,7 @@ export const labelAddComment = 'Add a comment';
 export const labelCommentAdded = 'Comment added';
 export const labelDisplayEvents = 'Display events';
 export const labelExportToPng = 'Export to png';
+export const labelGoToPerformancePage = 'Performance page';
```
Comment (Thebarda) [inline www/front_src/src/Resources/translatedLabels.ts:193]: Why not `Go to performance page` ?

Comment (jdelpierre) [reply www/front_src/src/Resources/translatedLabels.ts:193]: I don't know, you think is better to write this?

Comment (Thebarda) [reply www/front_src/src/Resources/translatedLabels.ts:193]: I mean, your label is labelGoToPerformancePage, that's why it confused me

Comment (jdelpierre) [reply www/front_src/src/Resources/translatedLabels.ts]: I push a new commit to change the name of variable ;)

File: www/front_src/src/Resources/Graph/Performance/index.tsx [hunk 2]
```diff
@@ -394,40 +427,56 @@ const PerformanceGraph = ({
...
             </Typography>
-            <div className={classes.exportToPngButton}>
-              <ContentWithCircularLoading
-                alignCenter={false}
-                loading={exporting}
-                loadingIndicatorSize={16}
-              >
-                <>
+            <ButtonGroup className={classes.buttonGroup} size="small">
+              <Button className={classes.buttonLink}>
```
Comment (Thebarda) [inline www/front_src/src/Resources/Graph/Performance/index.tsx:431]: You can use IconButton from centreon-frontend

Comment (jdelpierre) [reply www/front_src/src/Resources/Graph/Performance/index.tsx:431]: Ok, but the href isn't present on the IconButton ?

Comment (Thebarda) [reply www/front_src/src/Resources/Graph/Performance/index.tsx:431]: In your function, you can use history.push from useHistory

Comment (Thebarda) [reply www/front_src/src/Resources/Graph/Performance/index.tsx:431]: https://reactrouter.com/web/api/Hooks

Comment (jdelpierre) [reply www/front_src/src/Resources/Graph/Performance/index.tsx]: Ok thanks, I push a commit with your advices 👍

File: www/front_src/src/Resources/translatedLabels.ts [hunk 3]
```diff
@@ -190,6 +190,7 @@ export const labelAddComment = 'Add a comment';
 export const labelCommentAdded = 'Comment added';
 export const labelDisplayEvents = 'Display events';
 export const labelExportToPng = 'Export to png';
+export const labelPerformancePage = 'Performance page';
```
Comment (bdauria) [inline www/front_src/src/Resources/translatedLabels.ts:193]:
```suggestion
export const labelPerformancePage = 'Go to performance page';
```

Comment (jdelpierre) [reply www/front_src/src/Resources/translatedLabels.ts]: ok

**Comment by bdauria** [comment]
> Add a new button to access performance page from resource status detail panel, graph section.
> 
> Refs: MON-10900
> 
> Screenshot with the button on the right panel
> image
> 
> ## Type of change
> * [ ]  Patch fixing an issue (non-breaking change)
> * [x]  New functionality (non-breaking change)
> * [ ]  Breaking change (patch or feature) that might cause side effects breaking part of the Software
> * [ ]  Updating documentation (missing information, typo...)
> 
> ## Target serie
> * [ ]  20.04.x
> * [ ]  20.10.x
> * [x]  21.04.x
> * [x]  21.10.x (master)
> 
> ## How this pull request can be tested ?
> Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.
> 
> Any **relevant details** of the configuration to perform the test should be added.
> 
> ## Checklist
> * [ ]  I have followed the **coding style guidelines** provided by Centreon
> * [ ]  I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (_**docblock**_)
> * [ ]  I have commented my code, especially **hard-to-understand areas** of the PR.
> * [ ]  I have made corresponding changes to the **documentation**.
> * [ ]  I have **rebased** my development branch on the base branch (master, maintenance).

@jdelpierre I was wondering if we could try to shift the title to maintain it centered within the panel width?

**Comment by jdelpierre** [comment]
> I was wondering if we could try to shift the title to maintain it centered within the panel width?

Hum, Yes, I test to change the CSS props on the line concerned

**Comment by jdelpierre** [comment]
I have change CSS props to align the title
image

**Review by bdauria — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]
LGTM for translation

**Pull request merged**